tests/helpers.py

```diff
@@ -0,0 +1,9 @@
+from deepdiff import DeepDiff
+
+
+def compare_dicts(dict1, dict2):
```

Comment:
Minor, but how about giving a hint of the return value type already in the method name?
```suggestion
def area_dicts_identical(dict1, dict2):
```

Reply:
Or you add the assert already here, just like `_compare_bills `

Reply:
I think that I prefer your second solution for uniformity and rename these methods to `assert_dicts_identical` and `assert_lists_contain_same_elements`. Fixing it